src/main/java/com/nike/cerberus/operation/rds/RestoreRdsSnapshotOperation.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2019 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.operation.rds;
+
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.regions.Regions;
+import com.nike.cerberus.command.rds.RestoreRdsSnapshotCommand;
+import com.nike.cerberus.domain.environment.Stack;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.service.CloudFormationService;
+import com.nike.cerberus.service.RdsService;
+import com.nike.cerberus.store.ConfigStore;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import java.util.Map;
+
+import static com.nike.cerberus.module.CerberusModule.ENV_NAME;
+
+/**
+ * Operation for CopyRdsSnapshotsCommand
+ *
+ * Copies rds cluster snapshots in the primary for this environment to the secondary regions
+ */
+public class RestoreRdsSnapshotOperation implements Operation<RestoreRdsSnapshotCommand> {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private final ConfigStore configStore;
+    private final String environmentName;
+    private final RdsService rdsService;
+    private final CloudFormationService cloudFormationService;
+    private final String DESIRED_STATE = "available";
+    private final String COPYING_STATE = "copying";
+
+    @Inject
+    public RestoreRdsSnapshotOperation(ConfigStore configStore,
```

Comment:
Why is this command / operation needed at all, when we already have https://github.com/Nike-Inc/cerberus-lifecycle-cli/blob/master/src/main/java/com/nike/cerberus/operation/core/UpdateStackOperation.java#L51 which supports updating the DB stack?

Reply:
Nothing particular except for making it explicit and the environment variable override. I can take it out.